addons/mail/static/src/js/tours/mail.js

```diff
@@ -22,9 +22,12 @@ tour.register('mail_tour', {
         actions.text("SomeChannel_" + t, this.$anchor);
     },
 }, {
-    trigger: ".ui-autocomplete .ui-menu-item > a:contains(Create):has(.fa-lock)",
-    content: _t("<p> Create a private channel.</p>"),
```

Comment:
Shouldn't remove this from POT file?

Reply:
I have no clue, but it probably doesn't hurt to leave it to be safe. People who haven't yet (or will never) upgrade their db still need the old terms.

Reply:
ok